README update

MacHertZ233 opened the pull request:
Added Simplified Chinese translation.
Fixed minor problems.

r-value commented on README.md:
```diff
@@ -1,31 +1,33 @@
+ + Performance analysis of currently selected music / fumen
+   + Best tech score
+   + Best rating of current fumen
+   + Best Platinum Score / Star of current fumen
```
I don't think "Platinum Rating" should be removed 🤔

MacHertZ233 replied:
原来你指的是platinum rating，我还以为是跟上一项的rating重复了

r-value commented on README.md:
```diff
@@ -1,31 +1,33 @@
    + Rating coresponding to current tech score and badges in current gameplay
-   + Platinum Score / Star / Rating in minus style
-   + Achieved badges in current gameplay
+   + Current Platinum Score / Star in minus style
```
ditto

MacHertZ233 replied:
同上

r-value commented on README_chs.md:
```diff
@@ -0,0 +1,54 @@
+   + 获得的标记（包括FC/AP标记、FB标记和技术得分评级标记）
+   + 在玩家b50/r10/p50中的排名
+ + 实时显示游玩中谱面的成绩分析
+   + 当前技术得分（[类型-](#minus-style)）
```
maybe just use native markdown footnote here?

MacHertZ233 replied:
Perfect solution. Applied.

MacHertZ233 committed:
readme update

Added Simplified Chinese translation
Fixed missing and ambiguous info
Synced 4 commits from upstream and possibly still accumulating

MacHertZ233 committed:
readme: add mu3-mods info

r-value commented on README.md:
```diff
@@ -58,3 +63,11 @@ Put game assembly `Assembly-CSharp.dll` to `lib/` and use `dotnet build`
+
+[^3]:Minus style is a scoring method that starts from the highest score and deducts corresponding points for mistakes.
+
+[^4]:Retry function is provided by [BetterGiveUp](https://www.rainycolor.org/package/7EVENDAYSHOLIDAYS/BetterGiveUp/), one of the [mu3-mods](https://gitea.tendokyu.moe/akanyan/mu3-mods) collection.
```
Actually I don't think we should include these in `README` of a mod. I'm not interested in promoting this mod to someone without prior knowledge of these.

Also please don't refer to other mods whose author may not wish to have their work publicly linked to without their consent.

MacHertZ233 replied:
Links excluded. Necessary information retained.

r-value commented on README_chs.md:
Thank you for your efforts. However, after careful consideration, I have decided not to include localized documentation at this time.

r-value reviewed (commented):
Thank you for your document update and the wording fixes LGTM. I will proceed with the merge once the previous feedback has been addressed.

MacHertZ233 committed:
readme: fix footnote

MacHertZ233 committed:
exclude chs readme
